src/nodes/NamespaceDeclaration.php

```diff
@@ -18,8 +18,12 @@ public function getQualifiedNameAsString(): string {
     if ($name instanceof NameToken) {
       return $name->getText();
     }
+    
+    if (!$name) {
```

Comment:
`if(null === $name)` or `if(!($name is nonnull))`

Reply:
Thanks! I'm not much of a fan of Yoda conditionals, so I've updated with `$name === null`.